AsyncDisplayKit/ASImageNode.mm

```diff
@@ -110,17 +109,11 @@ - (instancetype)initWithViewBlock:(ASDisplayNodeViewBlock)viewBlock didLoadBlock
   return nil;
 }
 
-- (void)dealloc
```

Comment:
Ok, tracked it down.  The `-dealloc` deletion by this commit is a mistake and is a result of loosing the context and commit that my original changes were applied to (332ab23).

Commit b2810edb0f52, which took place after 332ab23, is the commit that added this `-dealloc` code.

**_IMPORTANT_**: As @maicki says, this `-dealloc` code needs to go back in (if it isn't already).

For reference, the reason why this showed up as a delete is because I gave @appleguy a copy of `ASImageNode.mm` with my changes (relative to 332ab23) _directly_, and I suspect that @applyguy copied that file in to his repo at a "more current than 332ab23, and after b2810edb0f52" point in time.